csv2geojson

```diff
@@ -2,7 +2,7 @@
 
 var optimist = require('optimist'),
     fs = require('fs'),
-    sys = require('sys'),
+    sys = require('util'),
```

Comment:
Sorry, I went too fast to commit. I dug up a little bit and it looks like it is not used anymore.